newscoop/admin-files/articles/edit_info_box.php

```diff
@@ -26,7 +26,20 @@
       <dt><?php echo $translator->trans('Number'); ?></dt>
       <dd><?php p($articleObj->getArticleNumber()); ?></dd>
       <dt><?php echo $translator->trans('Created by'); ?></dt>
-      <dd><?php p(htmlspecialchars($articleCreator->getRealName())); ?></dd>
+      <dd><?php
+          if ($articleCreator->getRealName()) {
+              p(htmlspecialchars(sprintf('%s %s', $articleCreator->getRealName(), $articleCreator->getLastName(), $articleCreator->getUserName())));
```

Comment:
👍 

btw I will also change it in the list view, like that:

![zrzut ekranu 2016-04-21 17 44 33](https://cloud.githubusercontent.com/assets/562536/14715348/c8aad00c-07e8-11e6-8656-8761f08d2623.png)

because its also displayed incorrectly, what do you think?